components/change_url_modal/change_url_modal.tsx

```diff
@@ -136,9 +144,9 @@ export default class ChangeURLModal extends React.PureComponent {
         return error;
     }
 
-    onSubmit = (e) => {
+    onSubmit = (e: React.MouseEvent<HTMLButtonElement>) => {
         e.preventDefault();
-        const url = this.refs.urlinput.value;
+        const url = this.urlInput.current ? this.urlInput.current.value : '';
```

Comment:
When I left this simply as `const url = this.urlInput.current.value;` Typescript gave me this error:
`TS2531: Object is possibly 'null'`

Not sure if there's a cleaner fix for this than the ternary expression I used?

Reply:
That seems fine to me.

Reply:
Yep same for me! I don't see a problem with this.

Reply:
Actually I just wanted to make an update. Currently our ESlinter doesn't allow this (not setup correctly for Typescript) however in such scenarios, Typescript best practices is to use the optional conditioning (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining). So in this case it would look something like 
`this.urlInput?.current?.value`.

Reply:
UPDATE! We have updated ESlint. You can now do
`const url = this.urlInput?.current?.value` and you should not see the `Object is possibly null` issue. Let me know how it goes!

Reply:
Optional conditioning... very cool! It worked out well.

Thanks for the docs link.